packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx

```diff
@@ -37,15 +37,15 @@ export interface IKeytipLayerState {
 const isMac = typeof navigator !== 'undefined' && navigator.userAgent.indexOf('Macintosh') >= 0;
 
 // Default sequence is Alt-Windows (Alt-Meta) in Windows, Option-Control (Alt-Control) in Mac
-const defaultStartSequence: IKeytipTransitionKey = {
+const defaultStart: IKeytipTransitionKey = {
```

Comment:
Why did you do all of this renaming? I get changing the things that are now private to be prefaced with '_' but other than that I don't really get renaming things to shorter versions of the strings...

Reply:
Just trying to make things less verbose/more concise. It's not required of course so I can undo if you think it's too much